### ISSUE TITLE
Excluded MX tags are hardcoded for TM2020

### ISSUE THREAD

**Issue by davidbmaier** [tm-rmc/MXRandom#57]
https://github.com/tm-rmc/MXRandom/blob/7cbbc86ae14a2245e8f6a980d97fc19aa407a0de/src/Utils/MX/Methods/LoadRandomMap.as#L92

These values work fine for TM2020, but on MP the tag IDs are different (see https://tm.mania.exchange/api/tags/gettags):
- 23 is "Remake"
- 37 is "Wood"
- 40 doesn't exist

Instead, shouldn't you be excluding 20 for "Kacky" ("Royal" and "Arena" don't exist on MP)?

**Comment by GreepTheSheep** [comment]
Yeah I'm excluding Kacky and Arena from tags for TM2020. But I forgot about MP.

For other map type than normal maps (Royal, FlagRush or Escape), I use another filtering (Map Type than Tags), which is better for filtering.

I just need to add a condition if we're on TM2020 or on Maniaplanet
